includes/definitions/discovery/uniping.yaml

```diff
@@ -0,0 +1,108 @@
+mib: DKSF-70-6-X-X-1
+modules:
+    sensors:
+        humidity:
+            data:
+                -
+                    oid: npRelHumTable
+                    value: npRelHumValue
+                    num_oid: .1.3.6.1.4.1.25728.8400.1.1.2.
+                    low_limit: npRelHumSafeRangeLow
+                    high_limit: npRelHumSafeRangeHigh
+                    skip_values: 0
+                    descr: 'Relative Humidity {{ $index }}'
+        state:
+            data:
+                -
+                    oid: npSmokeTable
+                    value: npSmokeStatus
+                    num_oid: .1.3.6.1.4.1.25728.8200.1.1.2.
+                    descr: 'Smoke {{ $index }}'
+                    skip_values: 5
+                    index: 'npSmokeStatus.{{ $index }}'
+                    state_name: npSmokeStatus
+                    states:
+                        - { descr: ok, graph: 0, value: 0, generic: 0 }
```

Comment:
These are a little easier to read if you put descr last ;)

Reply:
Our docs still have that way: https://docs.librenms.org/#Developing/Sensor-State-Support/

Reply:
haha, I didn't change all of them.  I only changed some.  I'll fix that up.